bin/prompt_prototype_upload_raws.sh

```diff
@@ -20,37 +20,47 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-# This script uploads the raw files from the HSC PDR2 run to Google Storage. It
-# renames the files to match prompt_prototype conventions. The user must have
-# gsutil already configured.
+# This script uploads the raw files from the HSC PDR2 run to a bucket at USDF.
+# It renames the files to match prompt_prototype conventions. The user must
+# have bucket access already configured.
 
 set -e  # Abort on any error
 
-RAW_DIR="/datasets/hsc/raw/ssp_pdr2/2016-03-07"
-UPLOAD_BUCKET=rubin-prompt-proto-unobserved
+RAW_DIR="/sdf/group/rubin/datasets/hsc/raw/ssp_pdr2/2016-03-07"
+UPLOAD_BUCKET=rubin-pp-users/unobserved
```

Comment:
Would it be possible to update the Buckets / USDF section of the Playbook to mention that `rubin-pp-users` is used for the unobserved files? There's analogous language in the Google Cloud section.